contact.html

```diff
@@ -30,6 +30,22 @@
       <a class="nav-toggler" target="_self" href="nav-mobile.html">
         <img class="icon" src="./icons/Enabled.svg" alt="hamburger menu" />
       </a>
+      <ul class="nav-ul">
```

Comment:
[OPTIONAL] This project is a one-pager. So it is not a multi-page. I think it would be better if you can use just the index.html file, otherwise your code is not DRY and some code pieces like header is repeating in each file.

Reply:
@huseyinbiyik  Thanks for everything, I have submitted the PR 4 times now you have noticed!😂. My other code reviewers didn't notice this thing before. Please say, how many times will I submit my same pr review?

Reply:
Hello @huseyinbiyik,
I am sorry for the previous comment. You were right, at that time I didn't understand your suggestions. Now I am in the third module and understand your speech.

Reply:
Dear @AbdusSattar-70,

Thank you for your message. It is so motivating to get this kind of feedback 👍  I hope you are enjoying the third module. Keep up the great work 🥳